ebean-api/pom.xml

```diff
@@ -39,6 +39,12 @@
       <artifactId>avaje-lang</artifactId>
       <version>1.0</version>
     </dependency>
+    
+    <dependency>
+      <groupId>io.avaje</groupId>
+      <artifactId>avaje-jsr305</artifactId>
+      <version>1.2</version>
+    </dependency>
```

Comment:
This dependency has provided scope in avaje-lang.
@rbygrave please check, if we really want to have this dependency here or if it should be fixed in avaje-lang instead